src/server.ts

```diff
@@ -29,9 +29,13 @@ async function createServer() {
       user: null,
     }),
     csrfPrevention: true,
-    // ADD ORIGIN FOR PROD AND STAGING
     cors: {
-      origin: [/http:\/\/localhost:/, 'https://studio.apollographql.com'],
+      origin: [
+        /http:\/\/localhost:/,
+        'https://studio.apollographql.com',
+        'https://paris2-0921.wns.wilders.dev',
+        'https://staging.paris2-0921.wns.wilders.dev/',
```

Comment:
If the url is `https://staging.paris2-0921.wns.wilders.dev/HOMEPAGE`, will it be allowed?
Is it needed to use a regex, like `/https:\/\/staging.paris2-0921.wns.wilders.dev/`?

Reply:
okay, will be fixed